models/prices/ethereum/prices_ethereum_tokens.sql

```diff
@@ -1790,7 +1790,8 @@ FROM
     ('gods-gods-unchained', 'ethereum', 'GODS', 0xccc8cb5229b0ac8069c51fd58367fd1e622afd97, 18),
     ('cah-moon-tropica', 'ethereum', 'CAH', 0x8e0e57dcb1ce8d9091df38ec1bfc3b224529754a, 18),
     ('bigtime-big-time', 'ethereum', 'BIGTIME', 0x64Bc2cA1Be492bE7185FAA2c8835d9b824c8a194, 18),
-    ('portal2-portal-coin', 'ethereum', 'PORTAL', 0x1bbe973bef3a977fc51cbed703e8ffdefe001fed, 18)
+    ('portal2-portal-coin', 'ethereum', 'PORTAL', 0x1bbe973bef3a977fc51cbed703e8ffdefe001fed, 18),
+    ('giv-giveth', 'ethereum', 'GIV', 0x900db999074d9277c5da2a43f252d74366230da0, 18)
```

Comment:
it appears this id value isn't valid. did you find the token on coinpaprika, grab the unique id on that page? can you double check and fix?

Reply:
Hey @jeff-dude The token is not listed yet, I just up the token address!
Sorry for the confusion, we have submitted the request and will update the PR when it's ready. Our token is a listed in LBank exchange and it probably will be listed there.

Reply:
@jeff-dude GIV is listed and I updated the id
https://coinpaprika.com/coin/giv-giv/

Reply:
nice -- looks good now!